### ISSUE TITLE
PostCSS Plugin Package

### ISSUE THREAD

**Issue by marcustisater** [postcss/postcss.com#18]
Something that we have mentioned before is building a "build you're own PostCSS package" section were you can get some kind of a postcss-package from just a few simpel clicks in a GUI similar as Yeoman.

@ai idea 
> You select basement: preprocessors or/and CSS4.
You select architecture: BEM, CSS Modules, etc.
You select a builder: Gulp, webpack, grunt, etc.
Pick some extra sugar.
We will get you a exactly code to copy-paste to your build process. For example, only gulp-cssnext with some sugar plugin additions if you select CSS4, BEM, Gulp and not so many additions.

**Comment by ai** [comment]
Maybe not only plugin package but also exact code examples for selected build tool.

Like wizard to add PostCSS to your project if you do not want to think :).

I can create a questions later.

**Comment by marcustisater** [comment]
Yeah, thanks. We need some psuedo to get this going before developing. 

Are we prioritizing this issue for first launch [1.0](https://github.com/postcss/postcss.com/milestones/1.0)?

**Comment by ai** [comment]
I think this issue can wait :-)

**Comment by marcustisater** [comment]
I like the one over at Zurb Foundation - very simple to use. http://foundation.zurb.com/sites/download.html/#customizeFoundation
